keps/sig-cloud-provider/providers/cloud-provider-baiducloud.md

```diff
@@ -0,0 +1,73 @@
+# Cloud Provider BaiduCloud
+
+## Table of Contents
+
+* [Table of Contents](#table-of-contents)
+* [Summary](#summary)
+* [Motivation](#motivation)
+    * [Goals](#goals)
+    * [Non-Goals](#non-goals)
+* [Requirements](#requirements)
+* [Proposal](#proposal)
+
+## Summary
+
+Baidu is a gold member of CNCF and we have a large team working on Kubernetes and related projects like complex scheduling, heterogeneous computing, auto-scaling etc. We build cloud platform to support Baidu emerging business including autonomous driving, deep learning, blockchain by leveraging Kubernetes. We also provide public container services named cloud container engine(CCE).
+
+## Motivation
+
+### Goals
+
+- Building, deploying, maintaining, supporting, and using Kubernetes on Baidu Cloud Container Engine(CCE) and Baidu Private Cloud(BPC). Both of the project are built on Kubernetes and related CNCF project.
+
+- Designing, discussing, and maintaining the cloud-provider-baidu repository under Github Kubernetes project. 
+
+### Non-Goals
+
+- Identify domain knowledge and work that can be contributed back to Kubernetes and related CNCF projects.
+
+- Mentor CCE and BPC developers to contribute to CNCF projects.
+
+- Provide an open platform for Baidu Kubernetes community to discuss Kubernetes related technologies and share knowledge.
```

Comment:
Was this supposed to be in `Goals` instead of `Non-Goals`?

Reply:
Revised.